### ISSUE TITLE
Passing Pseudo Parameters to dataSources Resource Causing Policy Failed

### ISSUE THREAD

**Issue by SupervisionT** [sid88in/serverless-appsync-plugin#336]
I'm trying to assignee a role for each dataSources using **iamRoleStatements**
When I pass the region as pseudo parameter **#{AWS::Region}** it failed and got the following error

An error occurred: GraphQlDsusersTableRole - The policy failed legacy parsing (Service: AmazonIdentityManagement; Status Code: 400; Error Code: MalformedPolicyDocument; Request ID: d24b4f63-9de9-410a-9af9-60e0c924fb85).

I have the same context for a lambda function and it works fine!

serverless-appsync-plugin version: 1.3.1

```
plugins:
  - serverless-pseudo-parameters
  - serverless-iam-roles-per-function
  - serverless-appsync-plugin

dataSources:
      - type: AMAZON_DYNAMODB
        name: usersTable
        description: Users Table
        config:
          tableName: {Ref: UsersTable}
          iamRoleStatements:
            - Effect: "Allow"
              Action:
                - "dynamodb:*"
                - "logs:CreateLogGroup"
                - "logs:CreateLogStream"
                - "logs:PutLogEvents"
              Resource:
                - "arn:aws:dynamodb:#{AWS::Region}:*:table/usersTable"
                - "arn:aws:dynamodb:#{AWS::Region}:*:table/usersTable/*"
```

**Comment by bboure** [comment]
@SupervisionT Are pseudo parameter working outside off appsync plugin?

Could you check the cloudformation output and check what is generated by the plugin? (under `.serverless/` directory.

**Comment by SupervisionT** [comment]
@bboure **Yes** pseudo parameter working outside off appsync plugin. I'm use the same expression in the lambda, roles & polices

I checked the cloudFormation output (under ```.serverless/```) and it didn't resolve the pseudo parameter!
The output is:
```
# cloudformation-template-update-stack.json

"Resource": [
                    "arn:aws:dynamodb:#{AWS::Region}:*:table/usersTable",
                    "arn:aws:dynamodb:#{AWS::Region}:*:table/usersTable/*"
]
```

**Comment by SupervisionT** [comment]
I did a debugging and It turns out that this issue is not related to the serverless-appsync-plugin.
The issue was in the order of executing the plugins in serveless!
So I changed the plugins sequence from:
```
plugins:
  - serverless-pseudo-parameters
  - serverless-iam-roles-per-function
  - serverless-appsync-plugin
```
To =>
```
plugins:
  - serverless-iam-roles-per-function
  - serverless-appsync-plugin
  - serverless-pseudo-parameters
```
And it works!
It makes sense where sls execute the plugins in the same order listed in the plugins section so if pseudo came before appsync plugin then it'll not affect the appsync plugin outputs.
